docs/plans/plans-crowdloan-datahighway-kusama-tanganika.md

```diff
@@ -0,0 +1,232 @@
+
+Your unlocked, unbonded KSM must be connected to the Polkadot.js extension. This step is not necessary if you participate through a partner exchange.
+
+## What is the Tanganika Network?
+
+Tanganika Network will be a Kusama Parachain and the DataHighway's parachain canary network.
+
+DHX is Tanganika's native token that was migrated from the DataHighway standalone chain at **1:1**.
+
+### Future Steps
+
+Verified ID - It will be necessary to get a "verified id" so that the Kusama parachain is shown in green under the list of parathreads in Polkadot.js Apps
+
+## What is the Parachain Testnet of Tanganika Network?
+
+The Spreehafen Parachain Testnet on Rococo (ROC tokens).
+
+### Historic Steps
+
+* Spreehafen code was written using Substrate Parachain Template and by following the Substrate Cumulus Workshop
+* DataHighway standalone chain pallet code was migrated to the parachain code branch
+* DataHighway created a new [DataHighway-Parachain](https://github.com/DataHighway-DHX/DataHighway-Parachain) repository and migrated the parachain code branch
+* ROC tokens received from the [Rococo faucet](https://riot.im/app/#/room/#rococo-faucet:matrix.org) and paraID 2116 registered.
+* Startup of 2x ChaChaCha and 2x Rococo Validators and the 2x related Collators that were updated with code from ‘ilya/parachain-update' branch in an attempt to run Spreehafen parachain so the parachain connects to the relay chain and were ready for submission to ChaChaCha and Rococo
+* Submitted our Collator Wasm and Genesis and were Onboarded as a Parathread and then as a Parachain on Rococo in Polkadot.js Apps 
+
+## What is Tanganika Network's DHX Token
+
+DHX token is the key to participating in the IoT-related DAO, Inter-Chain Data Market, the LPWAN Roaming Hub, and associated cross-chain DApps.
+
+DHX token is the native token of Tanganika Network.
+
+### Token Utility
+
+* DAO Governance - To participating in DHX Mining you must be an active stakeholder and participate in community governance. For details, see the [10:1 Rule](https://dev.datahighway.com/docs/whitepapers/shortpaper#the-101-rule) in the DataHighway Shortpaper.
+* Collator Rewards
+* Cross-Chain DApps
+* Cross-Chain Payments
+* Inter-Chain Data Market
+* LPWAN Roaming Hub
+
+### Distribution
+
+The following is the proportion of the total supply of DHX that will be distributed as rewards:
+
+* 0.3% - Crowdloan Contributors
+  * This represents 1% of the 30,000,000 DHX in the DHX DAO Unlocked Reserves (Treasury endowment for DHX Mining Rewards), which comprise 30% of the DataHighway's total supply of 100,000,000 DHX 
+
+DHX Mining Power (mPower) Rewards equivalent to a 40% mPower boost (based on 24 month period) shall apply to tokens that are contributed to the crowdloan.
+
+Note: In the crowdloan, those that lock their tokens to help the Tanganika Network secure a parachain slot will need to lock them for up to 48 weeks in a Kusama parachain crowdloan in order to get DHX tokens in return.
```

Comment:
parachain crowdloan to get DHX tokens in return.

Reply:
is this change just to reduce the word count? i can't see how it's grammatically incorrect https://www.collinsdictionary.com/dictionary/english/in-order-to

Reply:
> is this change just to reduce the word count?

Either way is ok, as mentioned I just proposed the cosmetic change. All ok